doc/howto/gh-actions-badges.md

```diff
@@ -0,0 +1,84 @@
+# Use GitHub Actions To Place A MyBinder.org Badge On Pull Requests
+
+You can facilitate review of GitHub pull requests on [mybinder.org](https://mybinder.org) by using [GitHub Actions](https://github.com/features/actions) to place a ![Binder](https://mybinder.org/badge_logo.svg) badge or link on Pull Requests.  Below are several examples of how to accomplish this.
+
+To enable GitHub Actions, you must place .yaml files that define your GitHub Action workflow in the `/.github/workflows/` directory in your repository.
+
+## Example 1: Comment on pull request with a binder badge
+
+_Note: this example will not work on pull requests from forks.  This is to protect repositories from malicious actors.  To trigger a GitHub Action with sufficient permissions to comment on a pull request from a fork, you must trigger the action via another event, such as a comment or a label.  This is demonstrated in Example 2 below._
+
+The below action uses the [github/script](https://github.com/actions/github-script) Action to call the [GitHub API](https://docs.github.com/en/rest/reference/issues#comments) for the purposes of making a comment on a PR that looks like this:
+
+> ![Binder](https://mybinder.org/badge_logo.svg) 👈 Launch a binder notebook on this branch
+
+```yaml
+#/.github/workflows/binder-badge.yaml
+name: Binder Badge
+on: [pull_request]
+
+jobs:
+  binder:
+    runs-on: ubuntu-latest
+    steps: 
+    - name: comment on PR with Binder link
+      uses: actions/github-script@v1
+      with:
+        github-token: ${{secrets.GITHUB_TOKEN}}
+        script: |
+          var BRANCH_NAME = process.env.BRANCH_NAME;
+          github.issues.createComment({
+            issue_number: context.issue.number,
+            owner: context.repo.owner,
+            repo: context.repo.repo,
+            body: `[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/${context.repo.owner}/${context.repo.repo}/${BRANCH_NAME}) :point_left: Launch a binder notebook on this branch`
+          }) 
+      env:
+        BRANCH_NAME: ${{ github.event.pull_request.head.ref }}
+```
+
+## Example 2: Comment with a Binder badge in response to a comment
+
+In the below example, we will trigger GitHub Actions to run after someone comments on a pull request with the command `/binder`, which will trigger Actions to comment on a PR in the same way as Example 1.
```

Comment:
maybe this should be `/binder-badge`? I feel like `/binder` might show up in, e.g., links and such. Or would it not be triggered if someone posted something like `myorg.com/binder`?

Reply:
I think this is only if there is an exact match, but still, what would we want as a command?

I like it when a flag/command describes whats done rather than just is a reference to something. But, there is no autocomplete etc for this so it should be easy and simple to remeber as well. Given that, perhaps /binder or /mybinder is a good call.

This is an example though so in the end anyone can adjust it for their repo.

Reply:
good point - if it's an exact match only, then I'm +1 on `/binder` and as you say, we can recommend other patterns in the future

Reply:
👍